src/EditorFeatures/Core/Implementation/Classification/SemanticClassificationUtilities.cs

```diff
@@ -165,56 +166,5 @@ internal static class SemanticClassificationUtilities
                 throw ExceptionUtilities.Unreachable;
             }
         }
-
-        private static async Task AddSemanticClassificationsAsync(
```

Comment:
This method and the method below were moved to `SemanticClassificationCacheUtilities.cs` so that they're accessible in the LSP layer.